duo_client/admin.py

```diff
@@ -1997,8 +1997,6 @@ def update_settings(self,
                         helpdesk_bypass_expiration=None,
                         helpdesk_message=None,
                         helpdesk_can_send_enroll_email=None,
-                        reactivation_url=None,
```

Comment:
In the (admittedly unlikely) scenario where someone is sending these parameters, this would be a breaking change requiring a major version bump.  So it might be best if we kept the method parameters and just document that they no longer do anything (see lines 2026-2030 as an example)/

so I'd suggest reverting these two lines and changing lines 2042-2044 to "Deprecated; ignored if specified" but all other deletions can be kept in place.